packages/optimistic-oracle/index.js

```diff
@@ -32,10 +33,11 @@ async function run({
   commonPriceFeedConfig,
   optimisticOracleProposerConfig,
   oracleType = "Voting",
+  optimisticOracleType = "OptimisticOracle",
```

Comment:
should probs verify the type passed in here to ensure it matches with either `OptimisticOracle` or `SkinnyOptimisticOracle`

Reply:
Done by changing to an enum